.github/workflows/cli-lint.yml

```diff
@@ -0,0 +1,77 @@
+name: CLI Lint
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'svix-cli/**'
```

Comment:
What about Rust lib changes? Since we're using a path dependency now, I think it would be good to test CLI on Rust lib changes.

Reply:
Sold